apps/glusterfs/examiner.go

```diff
@@ -0,0 +1,153 @@
+//
+// Copyright (c) 2018 The heketi Authors
+//
+// This file is licensed to you under your choice of the GNU Lesser
+// General Public License, version 3 or any later version (LGPLv3 or
+// later), or the GNU General Public License, version 2 (GPLv2), in all
+// cases as published by the Free Software Foundation.
+//
+
+package glusterfs
+
+import (
+	"fmt"
+	"reflect"
+
+	"github.com/boltdb/bolt"
+	"github.com/heketi/heketi/executors"
```

Comment:
Minor style nit: I've been trying to put a blank line between "external" imports and "intneral" imports even if there's a small number. If the list grows it prevents things from becoming a jumble.